src/application.js

```diff
@@ -5,9 +5,33 @@
 Marionette.Application = Marionette.Object.extend({
   cidPrefix: 'mna',
 
+  rootEl: 'body',
+
+  constructor: function(options) {
+    options = options || {};
+
+    this._initRootRegion(options);
+
+    Marionette.Object.prototype.constructor.apply(this, arguments);
+  },
+
+  _initRootRegion: function(options) {
+    var rootEl = options.rootEl || this.rootEl;
+
+    rootEl = Marionette._getValue(rootEl, this, options);
+
+    // rootEl is optional
+    if (rootEl) {
+
+      var RootRegion = Marionette.Region.extend({
+        el: rootEl
+      });
+
+      this.rootEl = new RootRegion();
```

Comment:
ninja edit.  Yeah I like that.  `showRootView` abstracts the region even more here.  :+1: :+1:   change coming soon